tags/create.tag

```diff
@@ -40,20 +44,32 @@
           if(!userProjectCollection){
             throw new Error('Error creating userProjectCollection');
           }
+          curProjectId = userProjectCollection.doc();
+
           var projectData = {
-            Project_Name : this.inputProjectTitle,
+            projectName : this.inputProjectTitle,
+            projectId: curProjectId.id
           };
-          userProjectCollection.add(projectData);
+          curProjectId.set(projectData);
+          console.log(curProjectId.id);
+
+          // trigger to pass curProjectId
+          observer.trigger('project:created', curProjectId.id);
 
           showDialog = false;
           this.refs.projectTitle.value = '';
           this.showProjectTitle = true;
+          this.showPickMonsters = true;
+
+
         }
         else{
           throw new Error('User is not signed in - should not see create tag');
         }
       }
     }
+    //add lisiner to pass project id to pickMonster tag
```

Comment:
@mml2222 did you mean to put in a listener here or is this an old comment that should be deleted?

Reply:
Yes, it an old comment! I will delete it